src/main/resources/application.yml

```diff
@@ -4,7 +4,7 @@ server:
   port: 8080
 torch:
   base:
-    url:
+    url: http://localhost
```

Comment:
Please look at the default. Also do we have an integration test were we can remove the Base URL in order to test whether the default works?